lmms_eval/tasks/docqa_hu_syn/utils.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+import logging
+
+from lmms_eval.tasks._task_utils.file_utils import generate_submission_file
+
+logger = logging.getLogger("lmms-eval")
+
+
+def docvqa_doc_to_visual(doc):
+    return [doc["image"].convert("RGB")]
+
+
+def docvqa_doc_to_text(doc, model_specific_prompt_kwargs):
+    question = doc["conversations"][0]["value"]
+    pre_prompt = model_specific_prompt_kwargs["pre_prompt"]
+    post_prompt = model_specific_prompt_kwargs["post_prompt"]
```

Comment:
The "question" text in both the IFEval and SynQA datasets already contain the `<image>` tag. Let's try to consolidate these differences before merging this in.